### ISSUE TITLE
Image thumbnail problems - Seems like a cache problem

### ISSUE THREAD

**Issue by para2022** [WWBN/AVideo#7692]
Hi,

Just recently my site has image problems. Here is whats happening.

1. No errors are reported in the logs on this.
2. Random images on the site appear blurred. If i save the same thumbnail again it fixes the issue and it displays how it should do.
3. If im logged out some vids show with no thumbnail (black). If i log back in the missing thumbs are back again. If i clear the cache and log back out the blank missing thumbs appear fixed and display correctly again.
4. When users uploads using direct upload, again the image is blurred, if i save the thumbnail again it then works again. It happens randomly.
5. This bug is not consistent but it is completely random and is happening quite regular throughout the site. This bug i have not encountered before and nothing was changed on the server other than git pull updates.

I suspect its a caching issue?

Thanks buddy

**Comment by para2022** [comment]
Fixed! By installing the encoder again. Not sure what happened? It suddenly created this problem out of the blue without me touching anything.

**Comment by para2022** [comment]
Yes its still happening throughout the site. I have reinstalled the encoder and that doesnt fix it.

**Comment by DanielnetoDotCom** [comment]
please give a link sample and also check your logs.

make sure you are in the latest git pull
